app/controllers/imports_controller.rb

```diff
@@ -1,17 +1,21 @@
 class ImportsController < ApplicationController
+  require "csv"
+
   include ActionView::Helpers::UrlHelper
   after_action :verify_authorized
 
   def index
     authorize :import
     @import_type = params.fetch(:import_type, "volunteer")
     @import_error = session[:import_error]
+    @sms_opt_in_error = session[:sms_opt_in_error]
```

Comment:
nit: the modal doesn't really represent an error. It's more like a warning or caution for the user.

Reply:
Renamed as a warning!